source/includes/linux/deploy-standalone.rst

```diff
@@ -1,11 +1,10 @@
 1) Download the MinIO Server
 ----------------------------
 
-The following tabs provide examples of installing MinIO onto 64-bit Linux
-operating systems using RPM, DEB, or binary. The RPM and DEB packages
-automatically install MinIO to the necessary system paths and create a
-``systemd`` service file for running MinIO automatically. MinIO strongly
-recommends using RPM or DEB installation routes.
+The following tabs provide examples of installing MinIO onto 64-bit Linux operating systems using RPM, DEB, or binary.
+The RPM and DEB packages automatically install MinIO to the necessary system paths and create a ``minio`` service for ``systemctl``.
```

Comment:
completely optional but it occurs to me we could just move this graf into the tab set and specify RPM or DEB respectively.

Then it only shows up when selecting the relevant tab. For binary builds, the user doesn't care about this information.

Reply:
After discussion, decided to leave it where it is so it's more visible. While the audience I had in mind wanted RPM/DEB and only got `systemctl` because we recommended it, there are other possible ways to trip over this.